src/main/java/com/auth0/json/mgmt/users/sessions/Session.java

```diff
@@ -0,0 +1,104 @@
+package com.auth0.json.mgmt.users.sessions;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Date;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Session {
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("user_id")
+    private String userId;
+    @JsonProperty("created_at")
+    private Date createdAt;
+    @JsonProperty("updated_at")
+    private Date updatedAt;
+    @JsonProperty("authenticated_at")
+    private Date authenticatedAt;
+    @JsonProperty("idle_expires_at")
+    private Date idleExpiresAt;
+    @JsonProperty("expires_at")
+    private Date expiresAt;
+    @JsonProperty("device")
+    private Device device;
+    @JsonProperty("clients")
+    private List<Client> clients;
+    @JsonProperty("authentication")
+    private Authentication authentication;
```

Comment:
last_interacted_at field is missing. For reference - https://auth0.com/docs/api/management/v2/users/get-sessions-for-user